palette/tests/hsluv_dataset/hsluv_dataset.rs

```diff
@@ -72,3 +80,22 @@ pub fn run_luv_to_xyz_tests() {
         assert_relative_eq!(to_xyz, v.xyz, epsilon = 0.001);
     }
 }
+
+#[test]
+pub fn run_lchuv_to_luv_tests() {
+    for (_, v) in TEST_DATA.iter() {
+        let to_luv: Luv<D65, f64> = v.lchuv.into_color_unclamped();
+        assert_relative_eq!(to_luv, v.luv, epsilon = 0.1);
+    }
+}
+
+#[test]
+pub fn run_luv_to_lchuv_tests() {
+    for (_, v) in TEST_DATA.iter() {
+        let mut to_lchuv: Lchuv<D65, f64> = v.luv.into_color_unclamped();
+        if to_lchuv.chroma < 1e-8 {
```

Comment:
This is a worth a comment.
In the implementation that generates the examples, `chroma` values below a certain threshold are a treated as 0.0, so the hue is set as 0 rather than using whatever comes out of `atan` (and probably isn't all that meaningful). My implementation matches the existing `Lab` -> `Luv` implementation and only does this if `u*` and `v*` are *strictly* 0. I can change my implementation to use a threshold on chroma instead, or we can leave this tweak in place in the test.

Reply:
There's going to be a point where the hue is almost just noise from the floating point operations. This was mentioned just yesterday in #107. I'm ok with a cut-off point like this.

Reply:
To be clear, do you want the cut-off point in the code itself for hue calculation, or just the tests?

Reply:
Ah, yes, sorry. Let's have it only in the test for now.